crates/forge/tests/cli/build.rs

```diff
@@ -42,6 +44,16 @@ contract Dummy {
 "#]].is_json());
 });
 
+forgetest!(initcode_size_exceeds_limit, |prj, cmd| {
+    prj.add_source("LargeContract", generate_large_contract(5450).as_str()).unwrap();
+    cmd.args(["build", "--sizes"]).assert_failure();
+});
+
+forgetest!(initcode_size_limit_can_be_ignored, |prj, cmd| {
+    prj.add_source("LargeContract", generate_large_contract(5450).as_str()).unwrap();
+    cmd.args(["build", "--sizes", "--ignore-eip-3860"]).assert_success();
+});
```

Comment:
Please also add assertions for stdout with `.stdout_eq(str![[r#""#]])` and then running the tests with `SNAPSHOT=overwrite` env

Reply:
I just pushed a commit adding the assertions.
The output we have right now displays a negative number telling by how many bytes the contract exceeds the size limit. But let me know if an error message instead of the table makes more sense.
Thank you!